package.json

```diff
@@ -396,9 +396,14 @@
         },
         "mssql.messagesDefaultOpen": {
           "type": "boolean",
-          "description": "true for the messages pane to be open by default; false for closed",
+          "description": "True for the messages pane to be open by default; false for closed",
           "default": true
         },
+        "mssql.resultsFontFamily": {
```

Comment:
should we also have a value to determine the font size?  I think now it's controlled by the editor size, which may be good enough, but an explicit option seems better in my opinion.